Utilities/Config.h

```diff
@@ -309,6 +312,73 @@ namespace cfg
 		}
 	};
 
+	// Float entry with custom Min/Max range.
+	template <s64 Min, s64 Max>
+	class _float final : public _base
+	{
+		static_assert(f64{Min} < f64{Max}, "Invalid cfg::_float range");
```

Comment:
Converting to f64 here makes no sense

Reply:
It's a narrowing conversion, it serves a second assertation.

Reply:
It's better to use s32 min/max then, not possible to trigger narrowing